Launcher/Classes/ViewControllers/TasksViewController.swift

```diff
@@ -239,7 +239,9 @@ class TasksViewController: NSViewController {
     }
     
     @IBAction func clickTagPicker(_ sender: Any) {
-        applicationStateHandler.tag = tagPicker.stringValue
+        if !tagPicker.stringValue.isEmpty {
+            applicationStateHandler.tag = tagPicker.stringValue
```

Comment:
Why is this needed? It doesn't seem like this would be the root of a crash. Setting a `UserDefault` to an empty `String` is perfectly valid.

Reply:
This doesn’t fix the crash, just a bad user experience, because this value doesn’t save. However, the sync part of this PR should fix one of the crashes (I’ve reproduce a crash once when the launcher was confused about bash output from mixed arguments)

Reply:
I think it is one of these (or both as both crashes reference to the same function): https://rink.hockeyapp.net/manage/apps/687667/app_versions/1/crash_reasons/203942335
https://rink.hockeyapp.net/manage/apps/687667/app_versions/1/crash_reasons/203947055

My theory is that because of async threads, it was performing faster and user was able to perform multiple click on Start button, that lead to crash as some stuff was still not finish for the first click.